.github/workflows/build.yml

```diff
@@ -31,8 +31,8 @@ jobs:
       - name: Search for go.mod and package.json files
         id: get-dirs
         run:
-          echo "dirs=$(find . \( -name 'go.mod' -o -name 'package.json' \) -exec dirname {} \; | sed
-          's|^\./||' | jq -Rsc 'split("\n")[:-1]')" >> ${GITHUB_OUTPUT}
+          echo "dirs=$(find . \( -name 'go.mod' -o -name 'package.json' -not -path '*/frontend/*' \)
```

Comment:
what was the behavior without this change? we use this in the product experience today, so would want to understand if it doesn't play well in a monorepo

Reply:
the problem was that without this change the build workflow was running `modus build` in the `rag/rag-mdfiles/frontend` directory (because there is a packakge.json file in that directory), however that directory is not a modus project, it's a ui project

Reply:
@jairad26 thoughts on solving this holistically?

Reply:
@ryanfoxtyler @jairad26 instead of looking for `package.json` or `go.mod` files could we just check for existence of `modus.json`?